lib/Model/Importer/Binary/MipsELFImporter.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+using namespace llvm;
+
+template <typename T, bool HasAddend>
+class MipsELFImporter {
+  llvm::Optional<uint64_t> SymbolsCount;
+  llvm::Optional<uint64_t> MIPSFirstGotSymbol;
+  llvm::Optional<uint64_t> MIPSLocalGotEntries;
+
+ public:
+  bool importMipsDynSymbol(TupleTree<model::Binary> &Model, uint64_t Tag) {
+    assert((Model->Architecture == model::Architecture::mips or
+            Model->Architecture == model::Architecture::mipsel) and
+           "The interface must be used for MIPS binaries only");
+    (void)Model;
```

Comment:
If the argument is unused, either drop it or just drop the name in the argument list.

Reply:
Even though it is being used within assertion only?

Reply:
Ah! Sorry, didn't notice this.
Then just use `revng_assert` and the problem will disappear. It automatically generates a `(void) sizeof(assertion)`.

Reply:
OK, thanks!